screengrab/lib/screengrab/page.html.erb

```diff
@@ -64,117 +64,99 @@
+        if (document.createEvent) {
+          var evObj = document.createEvent('MouseEvents', true);
+          evObj.initMouseEvent("click", false, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
+          el.dispatchEvent(evObj);
+        } else if (document.createEventObject) { //IE
+          var evObj = document.createEventObject();
+          el.fireEvent('onclick', evObj);
         }
+      }
 
-        for (index = 0; index < screenshotLink.length; ++index) {
-          screenshotLink[index].addEventListener('click', function(e) {
-            e.preventDefault();
+      for (index = 0; index < screenshotLink.length; ++index) {
+        screenshotLink[index].addEventListener('click', function(e) {
+          e.preventDefault();
 
-            var img = e.target;
-            if (e.target.tagName == 'A') {
-              img = e.target.children[0];
-            }
+          var img = e.target;
+          if (e.target.tagName == 'A') {
+            img = e.target.children[0];
+          }
 
-            // beautify
-            var tmpImg = new Image();
-            tmpImg.src = img.src;
-            imageDisplay.style.height     = 'auto';
-            imageDisplay.style.width     = 'auto';
-            if (window.innerHeight < tmpImg.height) {
-              imageDisplay.style.height = document.documentElement.clientHeight+'px';
-            } else if (window.innerWidth < tmpImg.width) {
-              imageDisplay.style.width = document.documentElement.clientWidth;+'px';
-            } else {
-              imageDisplay.style.paddingTop = parseInt((window.innerHeight - tmpImg.height) / 2)+'px';
-            }
+          // beautify
+          var tmpImg = new Image();
+          tmpImg.src = img.src;
+          imageDisplay.style.height     = 'auto';
+          imageDisplay.style.width     = 'auto';
+          if (window.innerHeight < tmpImg.height) {
+            imageDisplay.style.height = document.documentElement.clientHeight+'px';
+          } else if (window.innerWidth < tmpImg.width) {
+            imageDisplay.style.width = document.documentElement.clientWidth;+'px';
+          } else {
+            imageDisplay.style.paddingTop = parseInt((window.innerHeight - tmpImg.height) / 2)+'px';
+          }
 
-            imageDisplay.src             = img.src;
-            imageDisplay.alt             = img.alt;
-            imageDisplay.dataset.counter = img.dataset.counter;
+          imageDisplay.src             = img.src;
+          imageDisplay.alt             = img.alt;
+          imageDisplay.dataset.counter = img.dataset.counter;
 
-            imageInfo.innerHTML          = '<h3>'+img.alt+'</h3>';
-            imageInfo.innerHTML         += img.src.split("/").pop();
-            imageInfo.innerHTML         += '<br />'+tmpImg.height+'&times;'+tmpImg.width+'px';
+          imageInfo.innerHTML          = '<h3>'+img.alt+'</h3>';
+          imageInfo.innerHTML         += decodeURI(img.src.split("/").pop());
```

Comment:
Added `decodeURI` which was there in the iOS part. Correct me if this is wrong or whether it should be removed on both sides.